microraiden/microraiden/webui/js/main.js

```diff
@@ -4,7 +4,7 @@ function mainSwitch(id) {
   $(".container").show();
 }
 
-function pageReady(contractABI, tokenABI, startBlock) {
+function pageReady(contractABI, tokenABI) {
```

Comment:
all main.js changes needs to be rewritten, they're mostly a rollback to an old version, which wouldn't work on current version. Please, review each line of your changes to this file, against current master, and only include the changes you're sure you want to do (e.g. changes to comply with ui redesign).